portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +33,22 @@
 /** Servlet that sends and receives comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
nit: you may want to consider renaming "DataServlet" to use a more specific name.